diff --git a/lib/codemirror.js b/lib/codemirror.js
index 5ca96da4519458df1c52e871b2ae84566147e263..a1a08cd745ed4ab443dc24264e02c5604411f6d6 100644
--- a/lib/codemirror.js
+++ b/lib/codemirror.js
@@ -22,9 +22,10 @@
   var ie_upto10 = /MSIE \d/.test(userAgent);
   var ie_11up = /Trident\/(?:[7-9]|\d{2,})\..*rv:(\d+)/.exec(userAgent);
   var edge = /Edge\/(\d+)/.exec(userAgent);
+  var electron = /Electron\//.test(userAgent);
   var ie = ie_upto10 || ie_11up || edge;
   var ie_version = ie && (ie_upto10 ? document.documentMode || 6 : +(edge || ie_11up)[1]);
-  var webkit = !edge && /WebKit\//.test(userAgent);
+  var webkit = !edge && !electron && /WebKit\//.test(userAgent);
   var qtwebkit = webkit && /Qt\/\d+\.\d+/.test(userAgent);
   var chrome = !edge && /Chrome\/(\d+)/.exec(userAgent);
   var chrome_version = chrome && +chrome[1];
diff --git a/src/util/browser.js b/src/util/browser.js
index 751a06cd8be885d42c857ff6b3a66b5f89259841..9cae83cd2a12888e3099449589d76c9751a39f97 100644
--- a/src/util/browser.js
+++ b/src/util/browser.js
@@ -7,9 +7,10 @@ export let gecko = /gecko\/\d/i.test(userAgent)
 let ie_upto10 = /MSIE \d/.test(userAgent)
 let ie_11up = /Trident\/(?:[7-9]|\d{2,})\..*rv:(\d+)/.exec(userAgent)
 let edge = /Edge\/(\d+)/.exec(userAgent)
+let electron = /Electron\//.test(userAgent)
 export let ie = ie_upto10 || ie_11up || edge
 export let ie_version = ie && (ie_upto10 ? document.documentMode || 6 : +(edge || ie_11up)[1])
-export let webkit = !edge && /WebKit\//.test(userAgent)
+export let webkit = !edge && !electron && /WebKit\//.test(userAgent)
 let qtwebkit = webkit && /Qt\/\d+\.\d+/.test(userAgent)
 export let chrome = !edge && /Chrome\/(\d+)/.exec(userAgent)
 export let chrome_version = chrome && +chrome[1]
